localization/diagnostics/en.yaml

```diff
@@ -2967,6 +2967,16 @@
     value of type %0 has no property or method named 'subscript'; did you
     mean to use the subscript operator?
 
+- id: could_not_find_subscript_member_tuple
+  msg: >-
+    cannot access element using subscript for tuple type %0; 
+    use '.' notation instead
```

Comment:
Could you please remove line breaks in the diagnostic? Apparently LLVM YAML parser doesn't support folding yet, so it would still insert a newline :/

Reply:
No problem! Adjusted :)